### PR TITLE
global: celery 4.2 support for python 2.7

### LINKED ISSUE #53: global: celery v4.2.0 breaks task discovery
Because the extension is calling `celery.autodiscover_tasks` with an empty string (`''`) as the `related_name` argument, this [line eventually fails in celery](https://github.com/celery/celery/blob/9778f2bfe0bf0c4f3d554bdcb479ad769a349fc2/celery/loaders/base.py#L263) (responsible commit: https://github.com/WoaDmulL/celery/commit/c089f86e338c852043bc2ee25be2b82fd53a45a6).

Since it's not very convenient to change all of the invenio package entrypoints to support a `invenio_package` + `tasks` naming convention, there's two solutions:

1. Pin `celery` to `<4.2`
2. Wait until https://github.com/celery/celery/issues/4810 is resolved in upstream

### PR DESCRIPTION
**Pull request by lnielsen**
Closes #53

### CODE REVIEW

**Comment by lnielsen** [comment]
Seems like celery/celery#4810 will make it into 4.2.1 so let's wait for that - currently it's only broken for Python 2.7

**Comment by slint** [comment]
Maybe also use something like [`python_requires`](https://setuptools.readthedocs.io/en/latest/setuptools.html?highlight=python_requires#specifying-your-project-s-version) in the generated Invenio instance cookiecutter?

File: CHANGES.rst [hunk 1]
```diff
@@ -8,6 +8,16 @@
 Changes
 =======
 
+Version 1.0.1 (released 2018-06-13)
```
Comment (tiborsimko) [inline CHANGES.rst]: @lnielsen Is this fully release-ready? We should probably update the date to tomorrow then...  Can take care of releasing afterwards.

**Comment by lnielsen** [comment]
I removed the Celery 4.2 fixes since they are only applicable to Python 2.7, and cookiecutter for Invenio v3 is only for Python 3. Similarly, a fix is merged and will be released in Celery 4.3

**Pull request merged**